subcommand/inject-connect/command.go

```diff
@@ -145,10 +145,28 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	// create Consul API config object
+	cfg := api.DefaultConfig()
+	c.http.MergeOntoConfig(cfg)
+	if cfg.TLSConfig.CAFile == "" && c.flagConsulCACert != "" {
+		cfg.TLSConfig.CAFile = c.flagConsulCACert
+	}
+
+	// load CA file contents
+	var consulCACert []byte
+	if cfg.TLSConfig.CAFile != "" {
+		var err error
+		consulCACert, err = ioutil.ReadFile(cfg.TLSConfig.CAFile)
+		if err != nil {
+			c.UI.Error(fmt.Sprintf("Error reading Consul's CA cert file %s: %s", c.flagConsulCACert, err))
```

Comment:
```suggestion
			c.UI.Error(fmt.Sprintf("Error reading Consul's CA cert file %q: %s", c.flagConsulCACert, err))
```
I think having the filename quoted might make the error clearer that the filename is coming from an arg.